tee-worker/ts-tests/identity.test.ts

```diff
@@ -331,7 +343,8 @@ describeLitentry('Test Identity', (context) => {
         const substratePrimeIdentity = <LitentryIdentity>{
             Substrate: <SubstrateIdentity>{
                 address: `0x${Buffer.from(context.defaultSigner[0].publicKey).toString('hex')}`,
-                network: 'Litentry',
+                // When testing with integritee-node, change network to: TestNet
+                network: 'Litmus',
```

Comment:
See above.

If Integritee-node has a prefix - we can integrate it too.

What's better is we have a helper function in ts-test that gets the network from substrate node metadata (ss58Prefix) - similar to what we did in rust.

Reply:
The value is `13` when testing with integritee-node. Not sure whether this is their registered  ss58Prefix. Didn't check.  To have such a helper function is a nice idea. Will try.